### PR TITLE
Converting tests to page object model

### PR DESCRIPTION
**Pull request by dpford**
To be more in line with the way the rest of the bureau does testing, we now follow the page object model for our browser tests.  Each page is defined as an object with methods for each of the actions that would need to be performed on the page during the testing.  The layer on top of that is the step file which defines steps in plain english and maps those to methods on the page object.  The outermost layer is the feature files which use the gherkin style to define the features we want to test.

Instructions for running the tests are in the updated README.

### CODE REVIEW

**Comment by Scotchester** [comment]
Thanks, @dpford. This looks pretty good to me, as a testing know-nothing :) @orlandosoto, could you please review this when you get a chance?

**Comment by OrlandoSoto** [comment]
nice work @dpford ! can someone add me as collaborator to this repo so I can merge this?

**Comment by Scotchester** [comment]
That's okay, Orlando. Your :+1: is enough :)
